src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/MetadataTests/DefaultJsonTypeInfoResolverTests.JsonTypeInfo.cs

```diff
@@ -964,5 +964,142 @@ private class ClassWithExplicitOrderOfProperties
             [JsonPropertyOrder(-2)]
             public string A { get; set; }
         }
+
```

Comment:
Seems like the tests are in the wrong file? This is not testing `DefaultJsonTypeInfoResolver`.